src/content.ts

```diff
@@ -84,9 +72,24 @@ function listen(elem) {
     )
 }
 listen(window)
-document.addEventListener("readystatechange", _ =>
-    getAllDocumentFrames().forEach(f => listen(f)),
-)
+
+async function iframeMutationCallback(mutations: MutationRecord[], observer: MutationObserver) {
+    const leavegithubalone = await config.getAsync("leavegithubalone")
```

Comment:
I don't like that this function is async right now. The ideal thing would be to process all mutations synchronously and observe nested trees right away. Since `leavegithubalone` is not something that changes often, maybe we could cache it somewhere?

Reply:
You can use config.get if you want, that is synchronous. getAsync is merely more likely to be right :)